portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,35 @@
  */
 function addRandomGreeting() {
```

Comment:
nit: naming is not right since you are not adding anything you are replacing/updating the existing greeting using a random one. You probably want a name like "showRandomGreeting"